src/less/select/ds4/select.less

```diff
@@ -37,26 +28,42 @@ span.select__icon {
     }
 }
 
-.select--small {
-    font-size: @ds4-font-size-12;
+.select__control > input {
+    border-color: @ds4-color-core-gray-silver;
+    border-radius: 3px;
+    font-size: @ds4-font-size-14;
+    height: @ds4-height-control-medium;
+    padding: 0 36px 0 16px;
```

Comment:
Why did the padding on the right change to `36px` from `40px`?

Reply:
Oh, this wasn't a change, but it also has the `36px` instead of `40px` which has been there throughout.